tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml

```diff
@@ -0,0 +1,197 @@
+- _file_name: Aggregation_Dashboard_43.yaml
+  _roles:
+  - {{ SUPERSET_ROLES_MAPPING.instructor }}
+  dashboard_title: "Panel de agregaci\xF3n"
+  description: null
+  css: ''
+  slug: panel-de-agregacion
+  uuid: 51717a3d-6ff1-4b06-bbf5-7c07eadabc25
+  position:
+    DASHBOARD_VERSION_KEY: v2
+    DIVIDER-VFPZJjVlf2:
+      children: []
+      id: DIVIDER-VFPZJjVlf2
+      meta: {}
+      parents:
+      - ROOT_ID
+      - GRID_ID
+      type: DIVIDER
+    DIVIDER-lRzydt0Ybv:
+      children: []
+      id: DIVIDER-lRzydt0Ybv
+      meta: {}
+      parents:
+      - ROOT_ID
+      - GRID_ID
+      type: DIVIDER
+    DIVIDER-mG2gFzhZLS:
+      children: []
+      id: DIVIDER-mG2gFzhZLS
+      meta: {}
+      parents:
+      - ROOT_ID
+      - GRID_ID
+      type: DIVIDER
+    GRID_ID:
+      children:
+      - HEADER-m-cIE8RAmn
+      - TABS-Dc-1WI3Fle
+      - DIVIDER-lRzydt0Ybv
+      - HEADER-ASd1ryBTq4
+      - TABS-UMFjQI6rV3
+      - DIVIDER-VFPZJjVlf2
+      - DIVIDER-mG2gFzhZLS
+      id: GRID_ID
+      parents:
+      - ROOT_ID
+      type: GRID
+    HEADER-ASd1ryBTq4:
+      children: []
+      id: HEADER-ASd1ryBTq4
+      meta:
+        background: BACKGROUND_TRANSPARENT
+        headerSize: MEDIUM_HEADER
+        text: "Secci\xF3n 2: Datos espec\xEDficos"
+      parents:
+      - ROOT_ID
+      - GRID_ID
+      type: HEADER
+    HEADER-m-cIE8RAmn:
+      children: []
+      id: HEADER-m-cIE8RAmn
+      meta:
+        background: BACKGROUND_TRANSPARENT
+        headerSize: MEDIUM_HEADER
+        text: "Secci\xF3n 1: Datos globales"
+      parents:
+      - ROOT_ID
+      - GRID_ID
+      type: HEADER
+    HEADER_ID:
+      id: HEADER_ID
+      meta:
+        text: "Panel de agregaci\xF3n"
+      type: HEADER
+    ROOT_ID:
+      children:
+      - GRID_ID
+      id: ROOT_ID
+      type: ROOT
+    TAB-2Gg6rnNmk:
+      children: []
+      id: TAB-2Gg6rnNmk
+      meta:
+        defaultText: Tab title
+        placeholder: Tab title
+        text: "Comparaci\xF3n de cursos"
```

Comment:
Can you write the accents normally? or do they always need to be escaped?

Reply:
Those were exported like that, and the translation process also scape those: 

https://github.com/openedx/tutor-contrib-aspects/pull/242/files#diff-4a3d72c953a859f523f162da917e1141a825d2ef96d081a3e0732fb4d7d7d19cR34

Reply:
To align with the Aspects implementation, the text will be in English and proper translations will be added.